webpack.dev.config.js

```diff
@@ -4,9 +4,7 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 const autoprefixer = require('autoprefixer');
 
 module.exports = {
-  entry: {
-    app: resolve(__dirname, 'src/client/index.js'),
-  },
+  entry: ['@babel/polyfill', './src/client/index.js'],
```

Comment:
Also this one haha :grin:

Reply:
Is to Because fix regeneratorRuntime errors
Please visit [This web site](https://babeljs.io/docs/en/babel-polyfill) for more info